src/api/mediators.coffee

```diff
@@ -53,6 +55,18 @@ exports.addMediator = ->
 
   try
     mediator = this.request.body
+
+    if mediator?.endpoints?[0]?.host?
+      mediatorHost = mediator.endpoints[0].host
+    else
+      mediatorHost = 'unknown'
+
+    # audit mediator start
+    audit = atna.appActivityAudit true, mediator.name, mediatorHost, 'root'
```

Comment:
Yeah, I just put root there for lack of something better. `system` does sound better. I'l change it.